recipes.js

```diff
@@ -1,10 +1,109 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js');
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const data = require("./data.js");
 
-mongoose.connect('mongodb://localhost/recipeApp')
+mongoose
+  .connect("mongodb://localhost/recipeApp")
   .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
+    console.log("Connected to Mongo!");
+  })
+  .catch(err => {
+    console.error("Error connecting to mongo", err);
   });
+
+// The recipes.js file already connects to the recipesApp
+// database. Now we need to create a Recipe Schema.
+// The schema should have the following fields:
+
+const recipeSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  level: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"],
```

Comment:
for the level we want it to be an enum  

```
level: {
    type: String,
    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]
  } 
```

an enum is like a multiple choice